libs/react-helpers/src/lib/remove-0x.test.ts

```diff
@@ -0,0 +1,38 @@
+import { remove0x } from './remove-0x';
+
+test('It removes 0x from the start of a transaction hash', () => {
```

Comment:
question(non-blocking): when should we use test and when should we use it?

Reply:
Changed to use it and updated test name. Up for discussion if we should have guidelines on this. I've always just used whichever makes a more readable line.